assets/src/edit-story/components/inspector/prepublish/checklistTab.js

```diff
@@ -213,6 +227,47 @@ const ChecklistTab = (props) => {
           }
           ariaLabel={TEXT.RECOMMENDED_TITLE}
         >
+          {isVideoOptimizationEnabled && (
+            <VideoOptimizationGroup>
+              <PageIndicator>{__('General', 'web-stories')}</PageIndicator>
+              <DescriptionText>
+                {__(
+                  'Optimize all videos in the Story to ensure smooth playback.',
+                  'web-stories'
+                )}
+              </DescriptionText>
+              <ToggleGroup>
+                <StyledToggle
+                  id="automatic-video-optimization-toggle"
+                  checked={areVideosAutoOptimized}
+                  onChange={onAutoVideoOptimizationClick}
+                />
+                <DescriptionText
+                  forwardedAs="label"
+                  htmlFor="automatic-video-optimization-toggle"
+                >
+                  <TranslateWithMarkup
+                    mapping={{
+                      a: (
+                        <Link
+                          size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
+                          onClick={(evt) =>
+                            trackClick(evt, 'click_video_optimization_settings')
+                          }
+                          href={dashboardSettingsLink}
```

Comment:
I keep getting directed to a 404 page with this. 🤔

Reply:
It's working for me!